wolfssl/wolfcrypt/sp_int.h

```diff
@@ -686,6 +681,14 @@ typedef sp_int       mp_int;
  */
 typedef sp_int_digit mp_digit;
 
+/* Get the count of digits in the multi-precision number.
```

Comment:
This is now the same as what is in wolfmath.c
Remove the SP version of the function and re-enable the function for SP.

Reply:
OIC